android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/dashboard/DashboardFragment.java

```diff
@@ -131,11 +131,11 @@ private void initializePieChart(View root) {
 
         PieData pieData = new PieData(dataSet);
 
-        pieChart = root.findViewById(R.id.dashboard_piechart);
+        pieChart = binding.dashboardPiechart;
         pieChart.setDrawEntryLabels(false);
         pieChart.setData(pieData);
         pieChart.getDescription().setText("");
-        pieChart.setCenterTextSizePixels(12f);
+        pieChart.setCenterTextSize(12f);
         pieChart.getLegend().setTextSize(12f);
```

Comment:
okay not to use resources because if we do we might need to implement an sp to dp calculator.

Reply:
`setTextSize` already expects sp

Reply:
![image](https://user-images.githubusercontent.com/98540934/206904694-0fbde1e9-2a8d-4d68-8b52-1116bc0b9151.png)

sadly dp